velox/common/memory/MemoryArbitrator.cpp

```diff
@@ -45,6 +47,11 @@ std::unique_ptr<MemoryArbitrator> MemoryArbitrator::create(
       return nullptr;
     case Kind::kShared:
       return std::make_unique<SharedArbitrator>(config);
+    case Kind::kCustom:
```

Comment:
I'm trying to understand the "custom" kind. Is this a wildcard for all other implementations? Do we want to specify the exact names of these impls and let unsupported types fall to default section?

Reply:
A user cannot add entries to `MemoryArbitrator::Kind`. I would support removing `MemoryArbitrator::Kind` then use a std::string flag instead. But that would need for more changes.